class.jetpack-cli.php

```diff
@@ -922,6 +922,7 @@ public function sync( $args, $assoc_args ) {
 						'enqueue_wait_time'        => 0,
 						'queue_max_writes_sec'     => 10000,
 						'max_queue_size_full_sync' => 100000,
+						'full_sync_send_immediately' => true,
```

Comment:
Should you also add `full_sync_send_immediately_duration`?

Reply:
Might not be necessary

Reply:
I think we will need to revisit the entire cli